xyz.audiostellar.AudioStellar.yml

```diff
@@ -0,0 +1,344 @@
+      - type: archive
+        url: https://github.com/openframeworks/openFrameworks/releases/download/0.12.0/of_v0.12.0_linux64gcc6_release.tar.gz
+        sha256: 942cd299c903e0aeea5f05e429ab813702e0ba4f99a5a3d0e7c21e61be9b3b2b
+
+      - type: patch
+        path: patches/oF-ofAppGLFWWindow.cpp.patch
+      - type: patch
+        path: patches/oF-config.linux64.default.mk.patch
+        
+    # addons
+      - type: git
+        url: https://github.com/danomatika/ofxMidi.git
+        commit: f9d85fd888ba23cf49207b362e2bcc8cfad352ed
+        dest: addons/ofxMidi
+      - type: git
+        url: https://github.com/macramole/ofxJSON.git
+        commit: 8486eee1c752422a1ffd630656c814b2bb53305f
+        dest: addons/ofxJSON
+      - type: git
+        url: https://github.com/genekogan/ofxConvexHull.git
+        commit: 58a1a403e873189c9fb88d9b44988b11bc580821
+        dest: addons/ofxConvexHull
+      - type: git
+        url: https://github.com/macramole/ofxImGui.git
+        commit: 4d2dc03ae1b4c8138caaee4711c3a9c2c71bb7d3
+        dest: addons/ofxImGui       
+      - type: git
+        url: https://github.com/macramole/ofxAudioFile.git
+        commit: a5ed48298f7fa972f32c35e57e5e6d422f4581c5
+        dest: addons/ofxAudioFile
+      - type: git
+        url: https://github.com/macramole/ofxPDSP.git
+        dest: addons/ofxPDSP
+        commit: a001999a55fe1aa0a755fd8c608334882576f5af
+      - type: git
+        url: https://github.com/macramole/ofxAnn.git
+        dest: addons/ofxAnn
+        commit: 3957fdc4a5f56b5da378e1d92e4704212b98f5f2
+      - type: git
+        url: https://github.com/2bbb/ofxAbletonLink.git
+        dest: addons/ofxAbletonLink
+        commit: 4dd95c68337509d34785d0a65eaa7822213b41d6
+        disable-submodules: false
+    
+    cleanup:
+      - '*'
+
+
+  - name: audiostellar
+    buildsystem: simple
+    build-commands:
+      - mkdir -p bin/data
+      - cp -R assets bin/data/
+      - cp data-analysis/bin/* bin/
+      - OF_ROOT="$FLATPAK_DEST/lib/oF" make Release -j$FLATPAK_BUILDER_N_JOBS
+      - mkdir $FLATPAK_DEST/audiostellar
+      - cp -ar bin/* $FLATPAK_DEST/audiostellar
+      - ln -s ../audiostellar/audiostellar $FLATPAK_DEST/bin/audiostellar
+
+      - install -Dm644 xyz.audiostellar.AudioStellar.appdata.xml $FLATPAK_DEST/share/metainfo/xyz.ayrsd.AudioStellar.appdata.xml
+      - install -Dm644 xyz.audiostellar.AudioStellar.desktop $FLATPAK_DEST/share/applications/xyz.ayrsd.AudioStellar.desktop
+      - install -Dm644 assets/icon_256.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/xyz.audiostellar.AudioStellar.png
+    sources:
+      - type: git
+        url: https://gitlab.com/ayrsd/audiostellar.git
+        tag: "1.4"
+        commit: "9823b7c67dd01735974699cee6d6223f6655acd4"
+      
+      - type: file
+        path: xyz.audiostellar.AudioStellar.appdata.xml
+      - type: file
+        path: xyz.audiostellar.AudioStellar.desktop
```

Comment:
neither of these files is here. STILL.

Reply:
done in the last commit

Reply:
nope. that's the build failure.

Reply:
i'm sorry, i made a mistake, i've just commited the fix. thanks